src/eel2_adapter.h

```diff
@@ -42,6 +53,63 @@ class EEL2Adapter {
   double **mInputs = nullptr;
   double **mOutputs = nullptr;
 
+  World *mWorld;
+  Graph *mParent;
+
+  // cache the latest sndbuf b/c it is likely that we
+  // stick to one sndbuf
+  SndBuf *mSndBuf = nullptr;
+  int mSndBufNum = -1;
+
   // @todo make this a RT alloc char* or free this via NRT thread
   std::string mScript;
+
+  // see GET_BUF macro from SC_Unit.h
+  std::optional<SndBuf*> GetBuffer(int bufNum) {
```

Comment:
`std::optional<SndBuf*>` isn't idiomatic C++. Just return `nullptr` if the bufnum is out of range.

Reply:
Also, why PascalCase when most other methods are camelCase?